inference-engine/tests/functional/plugin/shared/src/single_layer_tests/lrn.cpp

```diff
@@ -36,15 +38,18 @@ std::string LrnLayerTest::getTestCaseName(testing::TestParamInfo<lrnLayerTestPar
 void LrnLayerTest::SetUp() {
     std::vector<size_t> inputShapes;
     auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    size_t alpha, beta, bias, size;
-    std::tie(alpha, beta, bias, size, netPrecision, inputShapes, targetDevice) = GetParam();
+    double alpha, beta, bias;
+    size_t size;
+    std::vector<size_t> axes;
+    std::tie(alpha, beta, bias, size, axes, netPrecision, inputShapes, targetDevice) = GetParam();
 
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
     auto params = ngraph::builder::makeParams(ngPrc, {inputShapes});
     auto paramIn =
         ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
 
-    auto lrn = std::make_shared<ngraph::opset1::LRN>(paramIn[0], alpha, beta, bias, size);
+    auto axes_node = std::make_shared<ngraph::op::Constant>(ngraph::element::i64, ngraph::Shape{axes.size()}, axes.data());
+    auto lrn = std::make_shared<ngraph::opset1::LRN>(paramIn[0], axes_node, alpha, beta, bias, size);
```

Comment:
Can you use `opset3` instead of `opset1`?

Reply:
Done